packages/driver/src/cy/assertions.ts

```diff
@@ -342,6 +344,7 @@ export const create = (Cypress: ICypress, cy: $Cy) => {
         }
 
         if (_.isFunction(onRetry)) {
+          //@ts-ignore
```

Comment:
options got more type safe, but completely and needs some more work here in the future.